test/controllers/sold.land.filter.controller.spec.ts

```diff
@@ -0,0 +1,53 @@
+jest.mock("ioredis");
+jest.mock("../../src/utils/application.data");
+
+import * as config from "../../src/config";
+import request from "supertest";
+import app from "../../src/app";
+import { SOLD_LAND_FILTER_PAGE_TITLE } from "../__mocks__/text.mock";
+import { ErrorMessages } from '../../src/validation/error.messages';
+
+describe("SOLD LAND FILTER controller", () => {
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe("GET tests", () => {
+    test(`renders the ${config.SOLD_LAND_FILTER_PAGE} page`, async () => {
+      const resp = await request(app).get(config.SOLD_LAND_FILTER_URL);
+
+      expect(resp.status).toEqual(200);
+      expect(resp.text).toContain(SOLD_LAND_FILTER_PAGE_TITLE);
+    });
```

Comment:
Add a simple unit test if you want to catch errors during the GET, something like
```
    test(`catch error when renders the ${config.SOLD_LAND_FILTER_PAGE} page on GET method`, async () => {
      mock_ANY_CALL_ON_GET_METHOD.mockImplementationOnce( () => { throw new Error(ANY_MESSAGE_ERROR); });
      const resp = await request(app).get(config.SOLD_LAND_FILTER_URL);

      expect(resp.status).toEqual(500);
      expect(resp.text).toContain(SERVICE_UNAVAILABLE);
    });
```